util/util.go

```diff
@@ -93,3 +96,22 @@ func GetDockerSocketPath() (string, error) {
 	}
 	return sockPath, nil
 }
+
+// GetPlatform returns the current platform we are running on by calling "python -mplatform"
+func GetPlatform() (string, error) {
+	cmd := exec.Command("python", "-m", "platform")
```

Comment:
Are you guaranteed to actually have python installed?

Reply:
I think python comes installed in most of the unix distributions, we can still add something to use `lsb_release` if the python command fails